breathecode/feedback/views.py

```diff
@@ -326,23 +326,22 @@ def get_review_platform(request, platform_slug=None):
         return Response(serializer.data)
 
 
-class ReviewView(APIView):
+@api_view(['GET'])
+@permission_classes([AllowAny])
+def get_reviews(request):
     """
     List all snippets, or create a new snippet.
     """
+    items = Review.objects.filter(is_public=True)
+    lookup = {}
```

Comment:
https://breathecode-test.herokuapp.com/admin/feedback/review/784/change/?_changelist_filters=status__exact%3DDONE this commentary is positive, but the grade ain't positive, it not have sense, investigate why